components/NavButtons/FAQ.tsx

```diff
@@ -1,11 +1,7 @@
 import Link from "next/link";
 import { Navigation } from "components/enum/navigation";
+import Button from "components/shared/Tailwind_Btn/TwButton";
 
 export const FAQButton = () => (
```

Comment:
```suggestion
export const FAQButton = () => <Button href={Navigation.FAQ} title="FAQ" />
```

nit - this could be one line :-)